java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2351,6 +2352,22 @@ public T objectRefcount(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Updates the last access time of specified <code>keys</code> and returns the count of keys that
+     * were updated.
+     *
+     * @apiNote When in cluster mode, the command may route to multiple nodes when <code>keys</code>
+     *     map to different <code>hash slots</code>.
```

Comment:
no need in this note here - transaction goes to a single node always

Reply:
https://github.com/aws/glide-for-redis/pull/1332#discussion_r1578641144

Reply:
Fixed.